### DISCUSSION TITLE
Sponsoring Clikt

### DISCUSSION THREAD

**Discussion by axhixh** [ajalt/clikt#560] [General]
My work has an open source fund and I want to nominate clikt this quarter. Is there a way to do this? 
Who can I talk to about this?

**Comment by ajalt** [comment]
I'm flattered that you want to sponsor me! But I'm fortunate enough to have a job that covers my bills, so I don't accept sponsorships. If you know of anyone who does want sponsorship to develop a feature in Clikt, I'd be happy to offer them guidance.

**Reply by axhixh** [reply]
I believe that Clikt and Mordant are two great libraries in Kotlin, not only does it help us build internal tools that can leverage the existing Kotlin code and the business logic in them, it also provides a great example of Kotlin code to learn from.

Thank you for writing and maintaining such great libraries.
